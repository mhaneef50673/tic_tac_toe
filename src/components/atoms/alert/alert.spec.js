import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Alert from './index';

describe('Alert Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <Alert
        type= "error"
        msg="test message"
      />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});