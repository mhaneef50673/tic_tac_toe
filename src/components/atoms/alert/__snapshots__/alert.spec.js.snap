// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert Component should render the component 1`] = `
<Alert
  msg="test message"
  type="error"
>
  <div
    className="alert error"
  >
    test message
  </div>
</Alert>
`;
